### ISSUE TITLE
Server Crash

### ISSUE THREAD

**Issue by jexline** [capsey/archeology-mod#19]
Hello, I tried the new file but get this error;

```
[01:44:47] [main/INFO]: Loading Minecraft 1.18.1 with Fabric Loader 0.12.11
[01:44:48] [main/WARN]: Warnings were found!
 - Mod 'Fabric Rendering Fluids (v1)' (fabric-rendering-fluids-v1) 0.1.18+3ac43d95c8 recommends any version of fabric-textures-v0, which is disabled for this environment (client/server only)!
	 - You should install any version of fabric-textures-v0 for the optimal experience.
[01:44:48] [main/INFO]: Loading 40 mods:
	- archeology 0.1.5
	- cloth-basic-math 0.6.0 via cloth-config
	- cloth-config 6.1.48
	- fabric 0.44.0+1.18
	- fabric-api-base 0.4.1+b4f4f6cdc8 via fabric
	- fabric-api-lookup-api-v1 1.4.0+16d92c47c8 via fabric
	- fabric-biome-api-v1 6.0.1+ded849a9c8 via fabric
	- fabric-command-api-v1 1.1.6+3ac43d95c8 via fabric
	- fabric-commands-v0 0.2.5+b4f4f6cdc8 via fabric
	- fabric-containers-v0 0.1.18+d154e2c6c8 via fabric
	- fabric-content-registries-v0 0.4.5+6f53a73dc8 via fabric
	- fabric-crash-report-info-v1 0.1.8+3ac43d95c8 via fabric
	- fabric-dimensions-v1 2.1.7+43d29571c8 via fabric
	- fabric-entity-events-v1 1.4.5+6b21378ac8 via fabric
	- fabric-events-interaction-v0 0.4.16+bfa23f17c8 via fabric
	- fabric-events-lifecycle-v0 0.2.6+b4f4f6cdc8 via fabric
	- fabric-game-rule-api-v1 1.0.10+3ac43d95c8 via fabric
	- fabric-item-api-v1 1.3.0+691a79b5c8 via fabric
	- fabric-item-groups-v0 0.3.3+3ac43d95c8 via fabric
	- fabric-lifecycle-events-v1 1.4.10+c15ca335c8 via fabric
	- fabric-loot-tables-v1 1.0.8+3ac43d95c8 via fabric
	- fabric-mining-level-api-v1 1.0.3+3ac43d95c8 via fabric
	- fabric-mining-levels-v0 0.1.7+b4f4f6cdc8 via fabric
	- fabric-networking-api-v1 1.0.18+3ac43d95c8 via fabric
	- fabric-networking-v0 0.3.5+b4f4f6cdc8 via fabric
	- fabric-object-builder-api-v1 1.11.0+3b82842ec8 via fabric
	- fabric-object-builders-v0 0.7.8+3ac43d95c8 via fabric
	- fabric-particles-v1 0.2.9+526dc1acc8 via fabric
	- fabric-registry-sync-v0 0.8.5+3ac43d95c8 via fabric
	- fabric-rendering-data-attachment-v1 0.3.4+7242e9d7c8 via fabric
	- fabric-rendering-fluids-v1 0.1.18+3ac43d95c8 via fabric
	- fabric-resource-loader-v0 0.4.11+3ac43d95c8 via fabric
	- fabric-screen-handler-api-v1 1.1.11+3ac43d95c8 via fabric
	- fabric-structure-api-v1 2.0.8+295197a7c8 via fabric
	- fabric-tag-extensions-v0 1.2.5+3ac43d95c8 via fabric
	- fabric-tool-attribute-api-v1 1.3.4+7de09f55c8 via fabric
	- fabric-transfer-api-v1 1.5.5+b4f4f6cdc8 via fabric
	- fabricloader 0.12.11
	- java 17
	- minecraft 1.18.1
[01:44:48] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.4 Source=file:/home/mch/multicraft/servers/server114936/libraries/net/fabricmc/sponge-mixin/0.10.7+mixin.0.8.4/sponge-mixin-0.10.7+mixin.0.8.4.jar Service=Knot/Fabric Env=SERVER
[01:44:48] [main/INFO]: Compatibility level set to JAVA_16
[01:44:52] [main/WARN]: @Inject(@At("INVOKE")) Shift.BY=3 on fabric-lifecycle-events-v1.mixins.json:server.WorldChunkMixin::handler$zcj000$onLoadBlockEntity exceeds the maximum allowed value: 0. Increase the value of maxShiftBy to suppress this warning.
[01:44:56] [main/INFO]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', servicesHost='https://api.minecraftservices.com', name='PROD'
[01:44:58] [main/WARN]: Ambiguity between arguments [teleport, location] and [teleport, destination] with inputs: [0.1 -0.5 .9, 0 0 0]
[01:44:58] [main/WARN]: Ambiguity between arguments [teleport, location] and [teleport, targets] with inputs: [0.1 -0.5 .9, 0 0 0]
[01:44:58] [main/WARN]: Ambiguity between arguments [teleport, destination] and [teleport, targets] with inputs: [Player, 0123, @e, dd12be42-52a9-4a91-a8a1-11c01849e498]
[01:44:58] [main/WARN]: Ambiguity between arguments [teleport, targets] and [teleport, destination] with inputs: [Player, 0123, dd12be42-52a9-4a91-a8a1-11c01849e498]
[01:44:58] [main/WARN]: Ambiguity between arguments [teleport, targets, location] and [teleport, targets, destination] with inputs: [0.1 -0.5 .9, 0 0 0]
[01:44:58] [main/INFO]: Reloading ResourceManager: Default, Fabric Mods (CapsLock Archeology Mod, Fabric Tool Attribute API (v1))
[01:44:58] [Worker-Main-32/ERROR]: Couldn't load tag archeology:clay_pots as it is missing following references: archeology:clay_pot (from CapsLock Archeology Mod)
[01:44:58] [Worker-Main-32/ERROR]: Couldn't load tag minecraft:dirt as it is missing following references: archeology:excavation_dirt (from CapsLock Archeology Mod)
[01:44:58] [Worker-Main-32/ERROR]: Couldn't load tag minecraft:moss_replaceable as it is missing following references: #minecraft:dirt (from Default)
[01:44:58] [Worker-Main-32/ERROR]: Couldn't load tag minecraft:lush_ground_replaceable as it is missing following references: #minecraft:moss_replaceable (from Default)
[01:44:58] [Worker-Main-32/ERROR]: Couldn't load tag minecraft:azalea_root_replaceable as it is missing following references: #minecraft:lush_ground_replaceable (from Default)
[01:44:58] [Worker-Main-32/ERROR]: Couldn't load tag minecraft:azalea_grows_on as it is missing following references: #minecraft:dirt (from Default)
[01:44:58] [Worker-Main-32/ERROR]: Couldn't load tag minecraft:enderman_holdable as it is missing following references: #minecraft:dirt (from Default)
[01:44:58] [Worker-Main-32/ERROR]: Couldn't load tag minecraft:bamboo_plantable_on as it is missing following references: #minecraft:dirt (from Default),#archeology:clay_pots (from CapsLock Archeology Mod)
[01:44:58] [Worker-Main-32/ERROR]: Couldn't load tag minecraft:big_dripleaf_placeable as it is missing following references: #minecraft:dirt (from Default)
[01:44:58] [Worker-Main-32/ERROR]: Couldn't load tag minecraft:mineable/shovel as it is missing following references: archeology:raw_clay_pot (from CapsLock Archeology Mod),archeology:excavation_dirt (from CapsLock Archeology Mod),archeology:excavation_gravel (from CapsLock Archeology Mod)
[01:44:58] [Worker-Main-32/ERROR]: Couldn't load tag archeology:excavation_blocks as it is missing following references: archeology:excavation_dirt (from CapsLock Archeology Mod),archeology:excavation_gravel (from CapsLock Archeology Mod)
[01:44:58] [main/WARN]: Failed to load datapacks, can't proceed with server load. You can either fix your datapacks or reset to vanilla with --safeMode
java.util.concurrent.ExecutionException: java.lang.IllegalStateException: Missing required tags: ResourceKey[minecraft:root / minecraft:block]:minecraft:azalea_grows_on,ResourceKey[minecraft:root / minecraft:block]:minecraft:azalea_root_replaceable,ResourceKey[minecraft:root / minecraft:block]:minecraft:bamboo_plantable_on,ResourceKey[minecraft:root / minecraft:block]:minecraft:big_dripleaf_placeable,ResourceKey[minecraft:root / minecraft:block]:minecraft:dirt,ResourceKey[minecraft:root / minecraft:block]:minecraft:enderman_holdable,ResourceKey[minecraft:root / minecraft:block]:minecraft:lush_ground_replaceable,ResourceKey[minecraft:root / minecraft:block]:minecraft:mineable/shovel,ResourceKey[minecraft:root / minecraft:block]:minecraft:moss_replaceable
	at java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:396) ~[?:?]
	at java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2073) ~[?:?]
	at net.minecraft.server.Main.main(Main.java:163) [server-intermediary.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:602) [fabric-loader-0.12.11.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77) [fabric-loader-0.12.11.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotServer.main(KnotServer.java:23) [fabric-loader-0.12.11.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.launch.server.FabricServerLauncher.main(FabricServerLauncher.java:66) [fabric-loader-0.12.11.jar:?]
Caused by: java.lang.IllegalStateException: Missing required tags: ResourceKey[minecraft:root / minecraft:block]:minecraft:azalea_grows_on,ResourceKey[minecraft:root / minecraft:block]:minecraft:azalea_root_replaceable,ResourceKey[minecraft:root / minecraft:block]:minecraft:bamboo_plantable_on,ResourceKey[minecraft:root / minecraft:block]:minecraft:big_dripleaf_placeable,ResourceKey[minecraft:root / minecraft:block]:minecraft:dirt,ResourceKey[minecraft:root / minecraft:block]:minecraft:enderman_holdable,ResourceKey[minecraft:root / minecraft:block]:minecraft:lush_ground_replaceable,ResourceKey[minecraft:root / minecraft:block]:minecraft:mineable/shovel,ResourceKey[minecraft:root / minecraft:block]:minecraft:moss_replaceable
	at net.minecraft.class_3505.method_33182(class_3505.java:55) ~[server-intermediary.jar:?]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:718) ~[?:?]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482) ~[?:?]
	at net.minecraft.class_4014.method_18365(class_4014.java:71) ~[server-intermediary.jar:?]
	at net.minecraft.class_4014.method_18367(class_4014.java:70) ~[server-intermediary.jar:?]
	at java.util.concurrent.CompletableFuture$UniCompletion.claim(CompletableFuture.java:572) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:714) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at net.minecraft.class_4014$1.method_18374(class_4014.java:53) ~[server-intermediary.jar:?]
	at net.minecraft.class_4014$1.method_18352(class_4014.java:50) ~[server-intermediary.jar:?]
	at java.util.concurrent.CompletableFuture$UniCompose.tryFire(CompletableFuture.java:1150) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1773) ~[?:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760) ~[?:?]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[?:?]
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?]
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?]
```

I uninstalled my other mods to try this with just fabric api and cloth config installed as well.

**Comment by Illest8** [comment]
Same here +1

**Comment by capsey** [comment]
Sorry for late response. Can you, please, try this one? @jexline @Illest8 

[server-fix-test.zip](https://github.com/capsey/archeology-mod/files/7852677/server-fix-test.zip)

**Comment by capsey** [comment]
Just uploaded update with the fix (v0.1.6). Hope this will help! Have fun playing @jexline @Illest8

**Comment by jexline** [comment]
Thank you! It works!
